app/openai/mcp_client.py

```
@@ -0,0 +1,145 @@
+                return result.content[0].text if result.content else None
+            except Exception as e:
+                print(f"Error fetching pet by ID: {e}")
+                return None
+
+    async def get_pets_by_owner(self, owner_id: int):
+        """소유자 ID로 펫 목록 조회"""
+        async with self.create_session() as session:
+            try:
+                result = await session.call_tool(
+                    "execute_sql",
+                    {"query": f"SELECT * FROM pet WHERE owner_id = {owner_id}"}
```

Ruff (0.12.2)

Possible SQL injection vector through string-based query construction

(S608)